src/inputType/PanInput.ts

```diff
@@ -149,9 +161,12 @@ export class PanInput implements IInputType {
 			if (!inputClass) {
 				throw new Error("Wrong inputType parameter!");
 			}
-			this.hammer = createHammer(this.element,
-				[Hammer.Pan, hammerOption],
-				inputClass);
+			this.hammer = createHammer(this.element, { ...{
+				recognizers: [
+					[Hammer.Pan, hammerOption],
+				],
+				inputClass,
+			}, ... this.options.hammerManagerOptions });
```

Comment:
I think `hammerManagerOptions` is somewhat ambiguous. 

In above comment, it looks `hammerManagerOptions` can have`{cssProps: {userSelect: "none",touchSelect: "none",touchCallout: "none",userDrag: "none"}. 

But in this code, you can replace recognizers, inputClass with new one `this.options.hammerManagerOptions`
```js
}, ... this.options.hammerManagerOptions });
```

Reply:
@jongmoon you're right. the hammerManager is that for advance users.
the structure of hammerManager options is so complex. if user want to change inputType or recognize, user can chage it.